kidviz/models.py

```diff
@@ -173,6 +173,18 @@ class Observation(TimeStampedModel, OwnerMixin):
 
     observation_date = models.DateField(default=now)
 
+    @property
+    def allowed_student(self):
```

Comment:
Same as in the other PR - since this returns multiple, function name should be plural